boards/nRF52840-DK/src/lib.rs

```diff
@@ -0,0 +1,580 @@
+//! Board support crate for the Nordic nRF52840-DK
+//!
+//! This crate is in early development. Not much to see here, right now.
+#![no_std]
+
+//#![deny(missing_docs)]
+#![deny(warnings)]
```

Comment:
I know `#![deny(warnings)]` is everywhere, but it's plain wrong and should be avoided. Detailed explanation and alternative here: https://github.com/rust-unofficial/patterns/blob/master/anti_patterns/deny-warnings.md

Reply:
Actually I am kinda sure I copied this from your crate too ;) Thanks for the hint tho; I will remove it! My OCD dictates me to remove all warnings anyways :D

Reply:
Yes, I'm as guilty as anyone :)

I'm going to remove these my crates as I'm working on them. So far most of them still have it.

Reply:
Fixed.